src/modules/pass/components/PassesDetail/PassesDetail.tsx

```diff
@@ -52,20 +56,34 @@ export const PassesDetail = () => {
   const { pass } = useFlags();
 
   const { data: tokenPass } = useGetPassById(tokenPassId);
+  const user = useSessionUser();
 
   const { mutate: registerUse } = usePostBenefitRegisterUse();
   const { data: benefits, isLoading: isLoadingBenefits } = useGetPassBenefits({ tokenPassId, chainId, contractAddress });
 
   const formatedData = useMemo(() => {
-    const data = benefits?.data?.items?.map((benefit) => {
+    const filteredBenefits = tokenPass?.data.tokenPassBenefits.filter(benefit => {
+      const isOperatorForBenefit = !! benefit.tokenPassBenefitOperators.find(operator => operator.userId === user?.id);
```

Comment:
```suggestion
      const isOperatorForBenefit = benefit.tokenPassBenefitOperators.some(operator => operator.userId === user?.id);
```

Reply:
Se entendi quando você faz o `!!` é para pegar o valor booleano, correto? Se for, só alterar por `.some` que ele faz isso.